qiskit/pulse/schedule.py

```diff
@@ -188,50 +192,46 @@ def _union(self, other: Tuple[int, ScheduleComponent]) -> 'Schedule':
         self._timeslots = self.timeslots.merge(sched_timeslots)
 
     def shift(self, time: int, name: Optional[str] = None) -> 'Schedule':
-        """Return a new schedule shifted forward by `time`.
+        """Return a new schedule shifted forward by ``time``.
 
         Args:
-            time: Time to shift by
-            name: Name of the new schedule. Defaults to name of self
+            time: Time to shift by.
+            name: Name of the new schedule. Defaults to the name of self.
         """
         if name is None:
             name = self.name
         return Schedule((time, self), name=name)
 
-    def insert(self, start_time: int, schedule: ScheduleComponent, buffer: bool = False,
+    def insert(self, start_time: int, schedule: ScheduleComponent,
                name: Optional[str] = None) -> 'Schedule':
-        """Return a new schedule with `schedule` inserted within `self` at `start_time`.
+        """Return a new schedule with ``schedule`` inserted into ``self`` at ``start_time``.
 
         Args:
-            start_time: Time to insert the schedule
-            schedule: Schedule to insert
-            buffer: Whether to obey buffer when inserting
-            name: Name of the new schedule. Defaults to name of self
+            start_time: Time to insert the schedule.
+            schedule: Schedule to insert.
+            name: Name of the new schedule. Defaults to the name of self.
         """
-        if buffer:
-            warnings.warn("Buffers are no longer supported. Please use an explicit Delay.")
         return self.union((start_time, schedule), name=name)
 
-    def append(self, schedule: ScheduleComponent, buffer: bool = False,
+    def append(self, schedule: ScheduleComponent,
                name: Optional[str] = None) -> 'Schedule':
-        r"""Return a new schedule with `schedule` inserted at the maximum time over
-        all channels shared between `self` and `schedule`.
+        r"""Return a new schedule with ``schedule`` inserted at the maximum time over
+        all channels shared between ``self`` and ``schedule``.
+
+        .. math::
 
-       $t = \textrm{max}({x.stop\_time |x \in self.channels \cap schedule.channels})$
+            t = \textrm{max}({x.stop\_time |x \in self.channels \cap schedule.channels})
```

Comment:
As discussed, the rendering of this is odd. Would be nice to figure this out but also maybe isn't a priority for this PR.